squid/docker-compose.yml

```diff
@@ -1,12 +1,43 @@
 version: "3"
 
 services:
+  node:
+    container_name: squid-node
+    build:
+      context: .
+      target: query-node
+    env_file: .env.example
+    environment:
+      DB_HOST: db
+    ports:
+      - "${GQL_PORT}:4350"
+    mem_limit: 300m
+    mem_reservation: 100m
+    command: "npm run query-node:start"
+    depends_on:
+      - db
+      - processor
+
+  processor:
+    container_name: squid-processor
+    build:
+      context: .
+      target: processor
+    env_file: .env.example
```

Comment:
We shouldn't use the example here, it's not future proof.

Reply:
This is an interesting point.

Taking into consideration that `docker compose` will be used only for local development purposes (I'm not pretending to use docker compose to build the image while deploying a new version of the app), how would you plan to define env variables locally?

My point is, we need to define the env variables first before running `make up` as [documented here](https://github.com/ChainSafe/Multix/blob/feature/squid-dockerization/squid/README.md#quickly-running-the-sample). So my question is, how do you plan to define these env variables before starting the app? We can leverage Makefile but I'm interested to know your position.

As soon as we have clear this way, I can change the docker-compose to use a proper `env_file` or define the `env` variables manually.

Reply:
well we can copy a local `.env` inside the docker, that's fine. The `.env.example` is only an example, and shouldn't be used anywhere. If the user want to use exactly these values, fine, then they copy/paste them into an `.env` that is never committed to GH, because it's their local setup.

Reply:
So the correct process is:

- Before running any docker command (or make command), set your environment variables first
- Run make build
- Run make up

If that's correct, then you are right, we can assume that an `.env` file should exists in the root of the squid folder and create a volume between your local machine and the docker container, so the app inside docker can access this file.

This is a good approach and we just need to improve the documentation. Let me know if you agree 👍🏼

Reply:
sorry for the late reply, yup I agree.

Reply:
Oh, sorry, I just made the commit to change that.